design-system/lib/blockies.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+// Copyright (c) 2019, Ethereum Name Service
+// The random number is a js implementation of the Xorshift PRNG
```

Comment:
Curious why we don’t use the [`blockies` NPM package](https://www.npmjs.com/package/blockies). Did I miss the explanation somewhere?

Reply:
@cloudhead made some improvements to it when he integrated it in the alt clients project and we're working off of that. It could probably be it's own package but for now we decided that this works.

Reply:
I think the blockies from that package don't to render the same way as the blockies everywhere else.